_ingest-pipelines/processors/fingerprint.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: default
+title: Fingerprint
+parent: Ingest processors
+nav_order: 55
+---
+
+# Fingerprint processor
```

Comment:
@gaobinlong Thank you for drafting this content. Doc review is completed. I made some edits for clarity. Please approve at your availability, and then I'll move this PR to editorial. Thank you.

Reply:
@vagimeli Thanks for your review, but I cannot approve this PR because I'm the author.